add new moengage source docs (#4127)
segmentio/segment-docs: src/connections/sources/catalog/cloud-apps/moengage-source/index.md

---
title: MoEngage Source
id: kpDbTUR9oD
beta: true
---

[MoEngage](https://www.moengage.com/){:target="_blank”} is an Intelligent Customer Engagement Platform. MoEngage allows brands to personalize every customer interaction and drive better engagement, retention, loyalty and lifetime value.

Using MoEngage as a source on Segment, you can evolve your team's engagement and marketing efforts. With MoEngage Streams, you can automatically forward your users' engagement and activity events to Segment. This will enable you to forward these events to your destinations or warehouses.

Segment allows you to move MoEngage data to your favorite BI or analytics tool in a seamless manner using which you can combine MoEngage data with the event data already flowing into Segment. This will help you get a holistic report of your engagement and marketing efforts.

This is an [Event Cloud Source](/docs/sources/#event-cloud-sources) which can export data into your Segment warehouse and federate the exported data into your enabled Segment destinations.

This source is maintained by MoEngage. For any issues with the source, [contact their Support team](mailto:support@moengage.com).

> _**NOTE:** The MoEngage Source is currently in beta, which means that they are still actively developing the source. If you are interested in joining their beta program or have any feedback to help improve the MoEngage Source and its documentation, [let their team know](mailto:support@moengage.com)._

Reviewer: ```suggestion
> info "This is a Beta source"
> The MoEngage Source is in beta, which means that they are still actively developing the source. If you're interested in joining their beta program or have any feedback to help improve the MoEngage Source and its documentation, [let their team know](mailto:support@moengage.com)._
```

| Onsite Message Template clicked | On clicking the Onsite Messaging template. | Campaign Activity | Web |
| Onsite Message Template shown | On a load of Onsite Messaging template. | Campaign Activity | Web |
| Onsite Message Template closed | On closing the Onsite Messaging template. | Campaign Activity | Web |
| Onsite Message Template auto dismiss | When the Onsite Messaging template closes by itself after the time duration given while creating the campaign. | Campaign Activity | Web |
| Card Clicked | Tracked when the user clicks the Card in the App Inbox. | Campaign Activity | Android, iOS |
| Card Delivered | Tracked when a Card is delivered to the App Inbox after the user has opened the Inbox. | Campaign Activity | Android, iOS |
| Card Sent | Tracked when a Card notification is sent to the user, doesn't indicate if the notification is delivered to the App Inbox. | Campaign Activity | Android, iOS |
| Card Viewed | Tracked when the user views a Card or scrolls down to it in the App Inbox. | Campaign Activity | Android, iOS |
| Connector Sent | Tracked when a connector was sent successfully. | Campaign Activity | Connector |

> | Email Bounced | Tracked when our system finds an email is hard bounced. Typically happens when an email address doesn't exist. | Campaign Activity | Email |

Reviewer 2: Is this on Segment or MoEngage side?
```suggestion
| Email Bounced | Tracked when MoEngage finds an email is hard bounced. Typically happens when an email address doesn't exist. | Campaign Activity | Email |
```

| Email Clicked | Tracked when a user clicks on any link in the email. | Campaign Activity | Email |
| Email Complained | Tracked when a user marks the email as spam. | Campaign Activity | Email |
| Email Dropped | Tracked when an email is dropped. Email is dropped, when it's part of the bounce list, the unsubscribe list, or the spam report list. | Campaign Activity | Email |
| Email Opened | Tracked when a user opens an email. | Campaign Activity | Email |

> | Email Sent | Tracked when our system sends an email to a user. | Campaign Activity | Email |

Reviewer 2: ```suggestion
| Email Sent | Tracked when MoEngage sends an email to a user. | Campaign Activity | Email |
```

> | Email Soft Bounced | Tracked when our system finds an email is soft bounced. Typically happens when there is a temporary delivery issue. | Campaign Activity | Email |

Reviewer 2: ```suggestion
| Email Soft Bounced | Tracked when MoEngage finds an email is soft bounced. Typically happens when there is a temporary delivery issue. | Campaign Activity | Email |
```

> | Email Unsubscribed | Tracked when our system finds an email is soft bounced. Typically happens when there is a temporary delivery issue. | Campaign Activity | Email |

Reviewer 2: ```suggestion
| Email Unsubscribed | Tracked when MoEngage finds an email is soft bounced. Typically happens when there is a temporary delivery issue. | Campaign Activity | Email |
```

| In-App Shown iOS | Tracked when an In-App message is shown to the user on an iOS device. | Campaign Activity | iOS |
| In-App Shown Windows | Tracked when an In-App message is shown to the user on a Windows device. | Campaign Activity | Windows |
| Notification Clicked Android | Tracked when a user clicks notification on an Android device. | Campaign Activity | Android |
| Notification Clicked iOS | Tracked when a user clicks notification on an iOS device. | Campaign Activity | iOS |
| Notification Clicked Web | Tracked when a user clicks notification on the browser. | Campaign Activity | Web |
| Notification Clicked Windows | Tracked when a user clicks notification on a Windows device. | Campaign Activity | Windows |
| Notification Dismissed iOS | Tracked when the user dismisses the notification on iOS device. | Campaign Activity | iOS |
| Notification Received Android | Tracked when a user receives a notification on an Android device. | Campaign Activity | Android |
| Notification Received iOS App Active | Tracked when App in foreground and user receives the notification on iOS. | Campaign Activity | iOS |

> | Notification Received iOS | Tracked when a user receives a notification on an iOS device. | Campaign Acitivity | iOS |

Reviewer 2: ```suggestion
| Notification Received iOS | Tracked when a user receives a notification on an iOS device. | Campaign Activity | iOS |
```

| Notification Sent iOS | Tracked when a user receives a notification on an iOS device. | Campaign Activity | iOS |
| Notification Swiped Android | Tracked when a notification is dismissed by the user with a swipe gesture on Android. | Campaign Activity | Android |
| On-site Message Shown | Tracked when an on-site messaging campaign is shown to a user. | Campaign Activity | Web |
| On-site Message Clicked | Tracked when an on-site messaging campaign is clicked by a user. | Campaign Activity | Web |
| On-site Message Closed | Tracked when an on-site messaging campaign is closed by a user. | Campaign Activity | Web |
| Push Preference Changed iOS | Tracked when iOS device push preference is changed. It has a modified status. | Campaign Activity | iOS |
| Web Personalization Message Shown | Tracked when a web personalization campaign is shown to a user. | Campaign Activity | Web |
| Web Personalization Message Clicked | Tracked when a web personalization campaign is clicked by a user. | Campaign Activity | Web |
| Web Personalization Message Closed | Tracked when a web personalization campaign is closed by a user. |  Campaign Activity | Web |

> | SMS Delivered | Tracked when our system receives a delivery receipt for the SMS sent to a user. | Campaign Activity | SMS |

Reviewer 2: ```suggestion
| SMS Delivered | Tracked when MoEngage receives a delivery receipt for the SMS sent to a user. | Campaign Activity | SMS |
```

> | SMS Sent | Tracked when our system sends SMS to a user. | Campaign Activity | SMS |

Reviewer 2: ```suggestion
| SMS Sent | Tracked when MoEngage sends SMS to a user. | Campaign Activity | SMS |
```

---
title: MoEngage Source
id: kpDbTUR9oD
beta: true
---

[MoEngage](https://www.moengage.com/) is an Intelligent Customer Engagement Platform. MoEngage allows brands to personalize every customer interaction and drive better engagement, retention, loyalty and lifetime value.

> Using MoEngage as a Source on Segment, you can evolve your team's engagement and marketing efforts. Leveraging MoEngage Streams, you can automatically forward your users' engagement and activity events to Segment. This will enable you to forward these events to your destinations or warehouses.

Reviewer 2: Current guidance from content team is that "sources" and "destinations" aren't capitalized
